src/commands/debugTestCommand.ts

```diff
@@ -1,9 +1,21 @@
+import { relative } from "path";
+import { workspace, WorkspaceFolder } from "vscode";
+
 import { getTestRunner } from "../runners/TestRunnerFactory";
 
-async function debugTest(rootPath, fileName, testName) {
+async function debugTest(
+  rootPath: WorkspaceFolder,
+  fileName: string,
+  testName: string
+) {
+  const relativeFilename = relative(String(rootPath), fileName);
   const testRunner = await getTestRunner(rootPath);
 
-  testRunner.debugTest(testName, fileName);
+  testRunner.debugTest(
+    workspace.workspaceFolders[0],
```

Comment:
This will always use first folder of a workspace as root folder, should be 

`workspace.getWorkspaceFolder(Uri.file(fileName))`

in order to pick up the correct root folder

Reply:
Indeed ! Good job 👍